services/submission/src/test/java/app/coronawarn/server/services/submission/validation/ValidSubmissionPayloadTest.java

```diff
@@ -16,17 +26,14 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.ActiveProfiles;
-import app.coronawarn.server.common.protocols.internal.SubmissionPayload;
-import app.coronawarn.server.services.submission.controller.ApiExceptionHandler;
-import app.coronawarn.server.services.submission.controller.RequestExecutor;
-import app.coronawarn.server.services.submission.controller.SubmissionPayloadMockData;
 
+/**
+ * This test must have the public modifier to be callable by the interpolation logic.
+ */
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @DirtiesContext
-@ActiveProfiles({"disable-ssl-client-verification",
-    "disable-ssl-client-verification-verify-hostname"})
-/* This test must have the public modifier to be callable by the interpolation logic */
-class ValidSubmissionPayloadTest {
+@ActiveProfiles({ "disable-ssl-client-verification", "disable-ssl-client-verification-verify-hostname" })
+public class ValidSubmissionPayloadTest {
```

Comment:
```suggestion
class ValidSubmissionPayloadTest {
```
Is it necessary to make the class public?